media-video/amdgpu-pro-amf/Manifest

```diff
@@ -2,3 +2,4 @@ DIST amdgpu-pro-amf-1.4.24.1452059.deb 1307216 BLAKE2B 2ecae593da961e2ff86e807c0
 DIST amdgpu-pro-amf-1.4.26.1511376.deb 1502528 BLAKE2B c2199f6b026552f486f2c7f75af9e3d37193870850de75d87b7099257281f98f030f4f7816aaf8086e0011a0f4db16210a084d4e7fcb8c3519e02be1ecb82de9 SHA512 3a3da593dfb765b7302e627a0df4c6e7bc35cfc76ce60ee9e5df74de73bcaf09b7dc687d30cd8ca54b25b90c140830bdb5352c816b7c6569f8200ca0f9acfc48
 DIST amdgpu-pro-amf-1.4.29.1580631.deb 1653852 BLAKE2B 5af5250c07e1b99ae8c1ee28507927e28ee7811929c789b5939a06f631b81155d6795e1392830945bf377f5298a5153e9af3541b2d94533661db394a059fb715 SHA512 07568c5e2f92f10274f7ac25a9d68539b8ce2e5d24c87e9b6a3aec2b8d5a32980c8fd802394dfa6eb421d3321dd60e07929e7f481578d23afcf623c5c9aa2fcc
 DIST amdgpu-pro-amf-1.4.30.1620044.deb 1671392 BLAKE2B 89611307dff4f77099ed52e8f7b523e842691b2fe970da7669a961048d5addd969068366a23d7c2df5710b67aeaf7a9cc3f4b902a708a4c686f00c32aa8cdb0c SHA512 4db5a814129a4d6e75466fd4e618eda155e031b70f184d5bf4dae78a13a61559fa26a1dcbc3d259064063467441822a34873af1142ab0366e38b61521c912bf4
+DIST amdgpu-pro-amf-1.4.31.1654522.deb 1694280 BLAKE2B ed2b6ab43b72bc7062bf7c5ec572983a9e3ebc96d2ed47d57903ebe12180a53cbc4848b81c9061da0aed30f32ec890b65cb75e234c1e942de6a277a5335d38bc SHA512 fdfc989f7dfcbe82081846b8cecd230fc927b1036a3cd5305e5299d843a4b3f19790f0f3be625322cb5cd79e2bf46721787843a52573baf65809196d6680f0f9
```

Comment:
I think we can safely drop the last part of this `${PV}` as there doesn't seem to be bumps between the minor versions.

Reply:
Actually there is, but I simply keep the latest "patch" with the minor:
examples:
https://repo.radeon.com/amdgpu/5.5.2/ubuntu/pool/proprietary/a/amf-amdgpu-pro/
https://repo.radeon.com/amdgpu/5.5.3/ubuntu/pool/proprietary/a/amf-amdgpu-pro/